### PR TITLE
Fix Loading configuration files section

### PR DESCRIPTION
**Pull request by gurelkaynak**
``$merge`` value is true by default, in the documentation I fixed it from true to false when a user wants to not overwrite existing configuration. When merge = true we should overwrite when merge = false we shouldn't overwrite, or am I missing a point?

### CODE REVIEW

**Comment by dereuromark** [comment]
According to https://github.com/cakephp/cakephp/blob/master/src/Core/Configure.php#L295 you are right :+1:

**Comment by dereuromark** [comment]
How about

> By keeping ``$merge`` to ``false``, values
  will not ever overwrite the existing configuration. Set it to ``true`` in order to replace the existing configuration.

**Comment by ravage84** [comment]
> By setting ``$merge`` to false, values will not ever overwrite the existing configuration.

If ``merge`` is ``false``, it will overwrite the existing values!

**Comment by markstory** [comment]
Both values of merge will overwrite scalar values. Are we talking about array values?

**Comment by gurelkaynak** [comment]
The doc is a bit confusing about this, even what I said is not 100% right but in plain english it makes more sense. This sentence: "By setting $merge to true, values will not ever overwrite the existing configuration." But it does. Need a better explanation. I will think about it and test the load function a bit more when I find some time.
